Config.cmake.in

```diff
@@ -0,0 +1,5 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/libtomoTargets.cmake")
+
+check_required_components(libtomo)
```

Comment:
```suggestion
@PACKAGE_INIT@

include("${CMAKE_CURRENT_LIST_DIR}/libtomoTargets.cmake")
include("${CMAKE_CURRENT_LIST_DIR}/libtomoTargetsVersion.cmake")

check_required_components(libtomo)

if(libtomo_FOUND)
    set(libtomo_LIBRARIES @VARIABLE_WITH_LIST_OF_TARGETS_IMPORTED_FROM_libtomoTargets@)
    
    # create an imported interface library which provides all the targets imported
    add_library(libtomo::libtomo INTERFACE IMPORTED)
    target_link_libraries(libtomo::libtomo INTERFACE ${libtomo_LIBRARIES})
endif()
```

Obviously, you don't want to apply this suggestion, because you need to create a variable before `configure_package_config_file` with the list of all the targets, e.g. `libtomo_TARGETS`, and replace `@VARIABLE_WITH_LIST_OF_TARGETS_IMPORTED_FROM_libtomoTargets@` with `@libtomo_TARGETS@`

Reply:
Thank you. I was having trouble undertanding the Config.cmake.in template. I think I did what you suggested. At least my test install has what seems like a more appropriate libtomoTargets-release.cmake.